domain/src/main/java/com/architect/codes/mu8/interactor/CharactersUseCaseImpl.kt

```diff
@@ -0,0 +1,4 @@
+package com.architect.codes.mu8.interactor
+
+class CharactersUseCaseImpl {
```

Comment:
Why do we have an empty use case?

Reply:
@recaldev  It was your UC, but I do not use it. I can delete it if you want.

Reply:
deleted